src/main/kotlin/com/terraformation/backend/customer/db/UserStore.kt

```diff
@@ -337,7 +337,11 @@ class UserStore(
   }
 
   fun updateGlobalRoles(userId: UserId, roles: Set<GlobalRole>) {
-    requirePermissions { updateGlobalRoles() }
+    if (roles.isEmpty()) {
```

Comment:
Can a TF-expert remove a 'super-admin' role from someone?

Reply:
No they should not be able to, if the roles are empty (remove all roles for user), the user must be able to `updateGlobalRoles` which is a blanket permission only given to "super admin".

Reply:
Got it.